lib/index.js

```diff
@@ -59,6 +52,17 @@ function startGame() {
   domUpdates.displayPlayerNames(playerNameInputs, domPlayerDivs);
 }
 
+function displayClue() {
+  clue = new Clue();
+  currentClue = currentRoundObj.clues[parseInt(event.target.dataset.id)];
+  // check if daily double
+  domUpdates.editClueHTML(currentClue, document.querySelector('.clue-text'));
+  domUpdates.changeDisplayedScreen(document.querySelector('.clue-pop-up'));
+  console.log(currentClue);
+  console.log(clue.validateAnswer(currentClue));
+  submitAnswer.addEventListener('click', (clue.validateAnswer.bind(null, currentClue)));
+}
+
```

Comment:
I think we might have both been working on this functionality today. I also wrote the displayClue() function. Sorry, I wasn't aware you were working on this but I completed this functionality today so maybe we should compare PRs.